lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -531,7 +531,16 @@ defmodule ExUnit.DocTest do
         raise Error,
           line: line_no,
           module: module,
-          message: "indentation level mismatch: #{inspect(line)}, should have been #{n_spaces}"
+          message: """
+          indentation level mismatch on doctest line: #{inspect(line)}
+
+          If you are planning to assert on the result of an `iex>` expression, \
+          make sure the result is indented at the beginning of `iex>`, which \
+          in this case is exactly #{n_spaces} spaces.
```

Comment:
```suggestion
          in this case is exactly #{n_spaces}.
```

Reply:
I find it less clear without ‘spaces’. And having the ‘spaces’ there certainly doesn’t seem to add confusion.

Reply:
@ecly The variable `n_spaces` contains the word "space" or "spaces". Right now is rendering "is exactly 2 spaces spaces"

Reply:
@fertapric haha, sorry. That went past me on my phone. Cheers